yt/data_objects/construction_data_containers.py

```diff
@@ -666,6 +669,9 @@ def __init__(
         self.level = level
         self.left_edge = self._sanitize_edge(left_edge)
         self.ActiveDimensions = self._sanitize_dims(dims)
+        self.fix_nonperiodic = [False, False, False]
+        self.lo_offset = np.array([0,0,0], dtype=int)
+        self.hi_offset = np.array([0,0,0], dtype=int)
```

Comment:
it's best practice to specify numpy dtypes more explicitly: here `"int64"` instead of builtin `int`